Update troubleshooting docs (#9044)
DataDog/documentation: content/en/tracing/profiler/profiler_troubleshooting.md

| RedHat                      | u262                                                           |
| Amazon (Corretto)           | u262                                                           |
| Bell-Soft (Liberica)        | u262                                                           |
| All vendors upstream builds             | u272                                                           |

If your vendor is not on the list, [open a support ticket][2], we can let you know if we're planning to support it or if we already offer beta support.

## Creating and using a JFR template override file

> Override templates help you override specific profiling properties. Care should be taken as the default settings are balanced for a good tradeoff between overhead and data density that should work well in most cases. In order to use an override file, perform the following steps:

Reviewer: ```suggestion
Override templates let you specify profiling properties to override. However, the default settings are balanced for a good tradeoff between overhead and data density that cover most use cases. To use an override file, perform the following steps:
```

> 1. Create an override file in a directory accessible by dd-java-agent at service invocation:

Reviewer: ```suggestion
1. Create an override file in a directory accessible by `dd-java-agent` at service invocation:
```

    ```
    touch dd-profiler-overrides.jfp
    ```

> 2. Add your desired overrides to the jfp file. For example, if you wanted to disable allocation profiling and JVM system properties, your `dd-profiler-overrides.jfp` file would look as following:

Reviewer: ```suggestion
2. Add your desired overrides to the jfp file. For example, to disable allocation profiling and JVM system properties, your `dd-profiler-overrides.jfp` file would look like the following:
```

    {{< code-block lang="text" filename="example-template.jfp" >}}
    jdk.ObjectAllocationInNewTLAB#enabled=false
    jdk.ObjectAllocationOutsideTLAB#enabled=false
    jdk.InitialSystemProperty#enabled=false
    {{< /code-block >}}

> 3. When running your application with `dd-java-agent`, your service invocation must point to the override file with `-Ddd.profiling.jfr-template-override-file=</path/to/override.jfp>`. Example invocation:

Reviewer: ```suggestion
3. When running your application with `dd-java-agent`, your service invocation must point to the override file with `-Ddd.profiling.jfr-template-override-file=</path/to/override.jfp>`, for example:
```
